nixos/tests/without-nix.nix

```diff
@@ -4,12 +4,6 @@ import ./make-test-python.nix ({ lib, ... }: {
     maintainers = [ ericson2314 ];
   };
 
-  nixpkgs.overlays = [
-    (self: super: {
-      nix = throw "don't want to use this";
-    })
-  ];
-
```

Comment:
@Ericson2314 note that the test fails if I add this to the actual machine configuration.

Reply:
Got nerd-sniped and fixed it.